redi-connect-front/src/components/molecules/ReadLanguages.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { Content } from 'react-bulma-components'
+import { RedProfile } from '../../types/RedProfile'
+import { connect } from 'react-redux'
+import { RootState } from '../../redux/types'
+import PipeList from './PipeList'
+import { Caption } from '../atoms'
+
+interface Props {
+  profile: RedProfile
+}
+
+const Placeholder = () => <Content italic>Input languages you speak here.</Content>
+
+const Me = ({ profile }: Props) => {
+  const {
+    languages
+  } = profile
+
+  if (!languages) return <Placeholder />
+
+  return languages && <PipeList items={languages} />
```

Comment:
Superfluous `languages &&`. `languages` is guarantees to be truthy here due to the return statement right above.